Source/Core/HeadingPitchRoll.js

```diff
@@ -0,0 +1,190 @@
+/*global define*/
+define([
+        './defaultValue',
+        './defined',
+        './DeveloperError',
+        './Math'
+    ], function(
+        defaultValue,
+        defined,
+        DeveloperError,
+        CesiumMath) {
+    "use strict";
+
+    /**
+     * A rotation expressed as a heading, pitch, and roll. Heading is the rotation about the
+     * negative z axis. Pitch is the rotation about the negative y axis. Roll is the rotation about
+     * the positive x axis.
+     * @alias HeadingPitchRoll
+     * @constructor
+     *
+     * @param {Number} [heading=0.0] The heading component in radians.
+     * @param {Number} [pitch=0.0] The pitch component in radians.
+     * @param {Number} [roll=0.0] The roll component in radians.
+     */
+    var HeadingPitchRoll = function(heading, pitch, roll) {
```

Comment:
Use the more concise `function HeadingPitchRoll` syntax here.  We use to use `var` to workaround a Jasmine bug that has been fixed.